test/e2e/e2e.go

```diff
@@ -46,3 +48,25 @@ func CreateRouteAndConfig(t *testing.T, clients *test.Clients, image string, opt
 	_, err := test.CreateRoute(t, clients, names)
 	return names, err
 }
+
+// WaitForScaleToZero will wait for the deployment specified by names to scale
+// to 0 replicas. Will wait up to 3 minutes before failing.
+func WaitForScaleToZero(t *testing.T, names test.ResourceNames, clients *test.Clients) {
+	t.Helper()
+	deploymentName := names.Revision + "-deployment"
```

Comment:
Use the function from here: https://github.com/knative/serving/blob/master/pkg/reconciler/v1alpha1/revision/resources/names/names.go#L23

Reply:
I'd even pull this out entirely and pass only the deployment name as a string. That narrows the API surface and makes the parameters more clear. WDYT?